README.md

```diff
@@ -63,3 +63,23 @@ Setting this up requires doing some initial Firebase setup, then doing some init
 
 ### Tests
 1. Run `npm run test` in the _functions_ directory to ensure that your changes haven't broken any of the functions.
+
+### Deploying slash command to multiple mattermost teams
```

Comment:
The way you've added this to the end means it appears to be a sub-section of "Developing".  

Instead, please move it up under "Finish Firebase Setup" and just add the text and skip the "Deploying slash command to multiple mattermost teams" section header.